Naggum.Backend/Matchers.fs

```diff
@@ -0,0 +1,7 @@
+﻿module Naggum.Backend.Matchers
+
+let (|String|Symbol|Object|List|) = function
+    | Reader.Atom (Reader.Object (:? string as s)) -> String s
```

Comment:
Thanks, friend, I've fixed that issue and added a test for it. Brilliant review!